kernel/malloc.c

```diff
@@ -230,38 +240,38 @@ static void * f_sbrk(int flags, int incr)
         heap_stack = &_stack - 4096;
     }
 
-    if (flags & MEM_USER) {
+    if ((flags & MEM_USER) == MEM_USER) {
```

Comment:
It's not a matter of preference, it's because a bit value is tested and not an integer, therefore the old version actually had a bug (tested while debugging).